<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string> Daily Task Notifier</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="headerLayout">
 <item>
  <widget class="QLabel" name="titleLabel">
   <property name="text">
    <string>üóìÔ∏è Daily Task Notifier</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
 </item>
 <item>
  <widget class="QLabel" name="subtitleLabel">
   <property name="text">
    <string>Organize, filter, and prioritize your daily tasks</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter</set>
   </property>
  </widget>
 </item>
</layout>
    </item>
    <item>
 <layout class="QHBoxLayout" name="searchLayout">
  <item>
   <widget class="QLineEdit" name="searchBar">
    <property name="placeholderText">
     <string>Search tasks (title, category, notes)...</string>
    </property>
    <property name="minimumSize">
     <size><width>300</width><height>32</height></size>
    </property>
   </widget>
  </item>
  <item>
   <widget class="QPushButton" name="searchButton">
    <property name="text">
     <string>Search</string>
    </property>
    <property name="minimumSize">
     <size><width>100</width><height>32</height></size>
    </property>
   </widget>
  </item>
 </layout>
</item>
    <item>
 <widget class="QGroupBox" name="filtersGroup">
  <property name="title">
   <string>Filters</string>
  </property>
  <layout class="QHBoxLayout" name="taskDetailsLayout">
   <property name="spacing">
    <number>8</number>
   </property>

   <item>
    <widget class="QLineEdit" name="categoryInput">
     <property name="placeholderText">
      <string>e.g., Work</string>
     </property>
     <property name="minimumSize">
      <size><width>140</width><height>32</height></size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="filterCategoryButton">
     <property name="text">
      <string>Filter by Category</string>
     </property>
     <property name="minimumSize">
      <size><width>140</width><height>32</height></size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="priorityInput">
     <property name="prefix">
      <string>Priority: </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
     <property name="minimumSize">
      <size><width>120</width><height>32</height></size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="filterPriorityButton">
     <property name="text">
      <string>Filter by Priority</string>
     </property>
     <property name="minimumSize">
      <size><width>140</width><height>32</height></size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dueDateInput">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size><width>140</width><height>32</height></size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="filterDateButton">
     <property name="text">
      <string>Filter by Date</string>
     </property>
     <property name="minimumSize">
      <size><width>140</width><height>32</height></size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="recurrenceInput">
     <property name="placeholderText">
      <string>e.g. Daily</string>
     </property>
     <property name="minimumSize">
      <size><width>140</width><height>32</height></size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="filterRecurrenceButton">
     <property name="text">
      <string>Filter by Recurrence</string>
     </property>
     <property name="minimumSize">
      <size><width>140</width><height>32</height></size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
</item>
    <item>
     <widget class="QPushButton" name="combinedFilterButton">
      <property name="text">
       <string>Apply Combined Filters</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sortByDateButton">
      <property name="text">
       <string>Sort by Due Date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sortByPriorityButton">
      <property name="text">
       <string>Sort by Priority</string>
      </property>
     </widget>
    </item>
    
    <item>
 <layout class="QHBoxLayout" name="mainContentRow">
  <property name="spacing">
   <number>12</number>
  </property>

  <!-- Left column: task list and view controls -->
  <item>
   <layout class="QVBoxLayout" name="taskListColumn">
    <item>
     <widget class="QListWidget" name="taskList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="viewControls">
      <item>
       <widget class="QPushButton" name="viewTodayButton">
        <property name="text">
         <string>View Today</string>
        </property>
        <property name="minimumSize">
         <size><width>120</width><height>34</height></size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="viewCompletedCheckBox">
        <property name="text">
         <string>View Completed Tasks</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </item>

  <!-- Right column: actions group -->
  <item>
   <widget class="QGroupBox" name="actionsGroup">
    <property name="title">
     <string>Actions</string>
    </property>
    <layout class="QVBoxLayout" name="actionsLayout">
     <item>
      <widget class="QPushButton" name="addTaskButton">
       <property name="text">
        <string>Add Task</string>
       </property>
       <property name="minimumSize">
        <size><width>160</width><height>36</height></size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export Current View</string>
       </property>
       <property name="minimumSize">
        <size><width>150</width><height>34</height></size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markCompletedButton">
       <property name="text">
        <string>Mark Selected as Completed</string>
       </property>
       <property name="minimumSize">
        <size><width>160</width><height>36</height></size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="archiveCompletedButton">
       <property name="text">
        <string>Archive Completed Tasks</string>
       </property>
       <property name="minimumSize">
        <size><width>150</width><height>34</height></size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetFiltersButton">
       <property name="text">
        <string>Reset Filters</string>
       </property>
       <property name="minimumSize">
        <size><width>220</width><height>36</height></size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </item>
 </layout>
</item>

   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
